packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -14,19 +14,29 @@ import uuid from 'uuid';
 import {InvalidUserCreation} from './convo-error';
 import ShareActivity from './share-activity';
 
+const idToUrl = new Map();
 
 const Conversation = WebexPlugin.extend({
   namespace: 'Conversation',
 
+  /**
+   * @param {Object} conversation
+   * @param {Object} object
+   * @param {Object} activity
+   * @returns {Promise}
+   */
   acknowledge(conversation, object, activity) {
+    const url = this.getConvoUrl(conversation);
+    const space = Object.assign({}, conversation, {url});
+
     if (!isObject(object)) {
       return Promise.reject(new Error('`object` must be an object'));
     }
 
-    return this._inferConversationUrl(conversation)
+    return Promise.resolve()
```

Comment:
Is server side HA coming at the same time as federation or is that a separate initiative? cc @yalo as this makes the bug you are researching not relevant.

Reply:
I was told the `url` in the DTO points towards the server-side HA version of the host. I see it points toward `conv-a` instead of `conv.a6` or `conv.a7`